web/impact/impact/v1/helpers/matching_program_criterion_helper.py

```diff
@@ -12,13 +12,21 @@
 class MatchingProgramCriterionHelper(MatchingCriterionHelper):
     application_field = "startup_id"
     judge_field = "expertprofile__home_program_family__name"
+    program_families = None
 
     def __init__(self, subject):
         super().__init__(subject)
         self._program_name_cache = None
 
+    @classmethod
+    def _program_family_map(cls):
+        if cls.program_families is None:
+            cls.program_families = cls.cache_instances_by_name(ProgramFamily)
```

Comment:
how does this behave if this class is loaded in memory and we were to add/remove program families?

Reply:
It would break. Fortunately, we do not add program families in the course of normal workflows. Could make an attempt to be robust about this, but YAGNI.